src/main/views/subscription-case-search.njk

```diff
@@ -0,0 +1,72 @@
+{% extends "template.njk" %}
+{% from "macros/common-components.njk" import submitButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "macros/common-components.njk" import goBack %}
+
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
```

Comment:
add {{super()}} call to this block to render user navigation

Reply:
added